### PR TITLE
Add documentation and test for new oeverify tool

### PR DESCRIPTION
**Pull request by jumaffre**
Now that Open Enclave's `host_verify` has been replaced with `oeverify` (0.12-rc1 release), I've updated the documentation and added a new test to verify node's quote with this tool.

Note that `/node/quote` used to return an array of one quote. I've changed this slightly so that it returns the quote info directly (slightly easier to parse).

### CODE REVIEW

File: tests/code_update.py [hunk 1]
```diff
@@ -4,153 +4,210 @@
 import infra.network
```
Comment (jumaffre) [inline tests/code_update.py:4]: Not too many changes in this file, mainly adding `test_verify_quotes` and wrapping the existing test in a new `test_update_all_nodes`.

File: doc/users/verify_quote.rst [hunk 2]
```diff
@@ -10,17 +10,22 @@ First, the client should connect to the node to verify, specifying the ``/node/q
     $ curl https://<ccf-node-address>/node/quote --cacert networkcert.pem
     {"quotes": [{"mrenclave":"<measurement_hash>, "node_id":<node_id>, "raw":"<hex_encoded_raw_quote>"}]}
 
-The raw quote should be decoded and output to file for verification via the Open Enclave ``host_verify`` command-line utility:
+The raw quote should be decoded and output to file for verification via the Open Enclave ``oeverify`` command-line utility:
 
 .. code-block:: bash
 
-    $ curl https://<ccf-node-address>/node/quote --cacert networkcert.pem | jq .quotes[0].raw | xxd -r -p > ccf_node_quote.bin
+    $ curl https://<ccf-node-address>/node/quote --cacert networkcert.pem | jq .raw | xxd -r -p > ccf_node_quote.bin
```
[the remark this answers is not in the document]
Comment (achamayou) [reply doc/users/verify_quote.rst:17]: Let's stick with quote for now, there's enough renaming going on at the moment, and there's no short term prospect of using anything other than SGX.

File: CHANGELOG.md [hunk 3]
```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 - Public tables in the KV must now indicate this in their name (with a `public:` prefix), and internal tables have been renamed. Any governance or auditing scripts which operate over internal tables must use the new names (eg - `ccf.members` is now `public:ccf.gov.members`).
 - `--member-info` on `cchost` can now take a third, optional file path to a JSON file containing additional member data (#1712).
+- `node/quote` endpoint now returns a single JSON object containing the node's quote (#1761).
```
Comment (achamayou) [inline CHANGELOG.md:17]: That needs to move up to Unreleased now.

File: CHANGELOG.md [hunk 4]
```diff
@@ -92,24 +93,24 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
-    
+
 ## [0.12.1]
 ### Changed
 - Release tarball replaced by a .deb
 
 ### Fixed
 - Fix LVI build for applications using CCF (#1466)
-    
+
```
Comment (achamayou) [inline CHANGELOG.md:113]: I would prefer we avoid reformatting unless we have an automatic formatter, such as markdownlint.

Comment (achamayou) [reply CHANGELOG.md:113]: https://github.com/DavidAnson/markdownlint looks like a plausible choice.

**Review by achamayou — APPROVED** [review]

**Pull request merged**